projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -237,3 +237,8 @@
     border-right: 7px solid transparent;
   }
 }
+
+.disabled {
+  opacity: 0.5;
+  pointer-events: none;
+}
```

Comment:
This seems unnecessary since disabled buttons already have their styling.

Reply:
I saw that, but that class is not applicable for label from the input field what I need in the case of checkbox field.

Reply:
I just saw that you were using this on a for-check div. You should move that disabled logic to input inside that div, like you did with the button. No need to set the class, just attribute disabled.

Reply:
Also tried that solution and there is nothing to emphasize user that the field is disabled.

In the below example Savings card is disabled, but how see that?

![Screenshot 2021-09-14 at 16 52 29](https://user-images.githubusercontent.com/75082466/133281217-a03efe98-e570-4636-8461-3d5e68fbbe0d.png)

Reply:
I see, then you can leave that class that you created, but remove `pointer-events` and perhaps rename it to `opaque` to be as generic as possible.

Reply:
I can't remove `pointer-events`, because then the label is clickable, and I should prevent that when the checkbox is disabled.

Reply:
No need to move pointer events. As I said in the template, you would use `disabled` attribute directly on an input element (that would actually disable the element natively instead with CSS) and perhaps add a class with only opacity as a rule to a wrapping div for styling. That's why I also suggested to resolve all that logic in the model, since then you would have a single property tied to that logic, that you could use on multiple places in the template.

Reply:
Managed to remove complex conditions for disable to model to the method I call on the select checkbox, so I'll remove `pointer-events`, rename class to `opaque` and add a disabled attribute to input field.